code.user.js

```diff
@@ -91,6 +92,23 @@
             this.inventoryUrlBase += '/';
     }
 
+    //#region language
+    var language = g_strLanguage;
+
+    var lang = { 'added to market for': 'added to market for', 'not added to market because': 'not added to market because', 'not added to market': 'not added to market', 'Could not retrieve the inventory': 'Could not retrieve the inventory', 'not turned into gems due to missing gems value': 'not turned into gems due to missing gems value', 'not turned into gems due to unknown error': 'not turned into gems due to unknown error', 'gems': 'gems', 'turned into': 'turned into', 'Processing': 'Processing', 'items': 'items', 'These items cannot be added to the market': 'These items cannot be added to the market', 'Item': 'Item', 'Items': 'Items', 'Sell': 'Sell', 'Buy': 'Buy', 'Sell All Items': 'Sell All Items', 'Sell Selected Items': 'Sell Selected Items', 'Turn Selected Items Into Gems': 'Turn Selected Items Into Gems', 'Sell All Cards': 'Sell All Cards', 'Reload Inventory': 'Reload Inventory', 'This is likely the highest buy order price': 'This is likely the highest buy order price', 'The best price is': 'The best price is', 'Search': 'Search', 'Loading market listings': 'Loading market listings', 'Deselect all': 'Deselect all', 'Select all': 'Select all', 'Remove selected': 'Remove selected', 'Relist selected': 'Relist selected', 'Relist overpriced': 'Relist overpriced', 'Select overpriced': 'Select overpriced', 'Remove selected 2': 'Remove selected', 'Unknown Item': 'Unknown Item', 'Number of items': 'Number of items', 'worth': 'worth', 'Select all from page': 'Select all from page', 'Calculate prices as the': 'Calculate prices as the', 'maximum of the average': 'maximum of the average', 'hours': 'hours', 'and lowest listing': 'and lowest listing', 'lowest listing': 'lowest listing', 'The value to add to the calculated price': 'The value to add to the calculated price', 'minimum and maximum are respected': 'minimum and maximum are respected', 'Use the second lowest listing when the lowest listing has a low quantity': 'Use the second lowest listing when the lowest listing has a low quantity', 'Minimum': 'Minimum', 'and maximum': 'and maximum', 'price for normal cards': 'price for normal cards', 'price for foil cards': 'price for foil cards', 'price for other items': 'price for other items', 'Market items per page': 'Market items per page', 'Automatically relist overpriced market listings': 'Automatically relist overpriced market listings', 'slow on large inventories': 'slow on large inventories' };
```

Comment:
While this works, I don't think this is ideal for maintenance (I also think it's pretty hard to read). I think it would work better if we used a separate file for each locale (for example, en.json, schinese.json) and then with an identifier instead of the full name (e.g., 'addedToMarketFor'). I think it should be possible to host these on GitHub (this way it's also possible to just update the languages). I welcome some additional input on this before merging this.